sonic-ycabled/proto_out/linkmgr_grpc_driver_pb2_grpc.py

```diff
@@ -0,0 +1,259 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
```

Comment:
Are these changes added by mistake? If so can you please remove?

Reply:
Removed